src/input_stream.c

```diff
@@ -83,7 +84,6 @@ static struct aws_input_stream_vtable s_external_input_stream_vtable = {
     .read = s_external_input_stream_read,
     .get_status = s_external_input_stream_get_status,
     .get_length = s_external_input_stream_get_length,
-    .destroy = s_external_input_stream_destroy,
```

Comment:
shouldn't you be overriding the `acquire/release` vtable functions, and having them call `aws_crt_resource_acquire()/aws_crt_resource_release()` under the hood?

since aws-crt-ffi has its own concept of ref-counting

Reply:
You are right... I thought ffi will not expose the real aws_input_stream, but under the hood, other C function still touch the aws_input_stream. So, it should be the refcount to the resource.